flink/v1.20/flink/src/test/java/org/apache/iceberg/flink/source/TestIcebergSourceFailover.java

```diff
@@ -76,6 +76,9 @@ public class TestIcebergSourceFailover {
   // The goal is to allow some splits to remain in the enumerator when restoring the state
   private static final int PARALLELISM = 2;
   private static final int DO_NOT_FAIL = Integer.MAX_VALUE;
+  // FIXME (before merge): should it be parameterized test or hardcoded value with some comment that
```

Comment:
@pvary WDYT? Should I extract parameter tests or keep this hardcoded to not produce too much tests?